Sources/Services/AWSS3/Sources/AWSS3/Plugins.swift

```diff
@@ -44,7 +45,7 @@ public class DefaultAWSAuthSchemePlugin: ClientRuntime.Plugin {
     public func configureClient(clientConfiguration: ClientRuntime.ClientConfiguration) throws {
         if let config = clientConfiguration as? S3Client.S3ClientConfiguration {
             config.authSchemeResolver = DefaultS3AuthSchemeResolver()
-            config.authSchemes = [AWSSDKHTTPAuth.SigV4AuthScheme(), AWSSDKHTTPAuth.SigV4AAuthScheme()]
+            config.authSchemes = [AWSSDKHTTPAuth.SigV4AuthScheme(), AWSSDKHTTPAuth.SigV4AAuthScheme(), AWSSDKHTTPAuth.SigV4S3ExpressAuthScheme()]
```

Comment:
*[Note: this is generated code]*

`SigV4S3ExpressAuthScheme` is added to the service's auth schemes